Utilities/build-script-helper.py

```diff
@@ -475,7 +329,8 @@ def build_llbuild_using_cmake(args, target, swiftc_exec, build_dir, base_cmake_f
         '-DCMAKE_C_COMPILER:=clang',
         '-DCMAKE_CXX_COMPILER:=clang++',
         '-DCMAKE_CXX_FLAGS=-target %s' % target,
-        '-DLLBUILD_SUPPORT_BINDINGS:=Swift'
+        '-DLLBUILD_SUPPORT_BINDINGS:=Swift',
```

Comment:
This looks like a typo

```suggestion
        '-DLLBUILD_SUPPORT_BINDINGS=Swift',
```